### ISSUE TITLE
ValueError: No input/output device matching 'default'

### ISSUE THREAD

**Issue by lukaszliniewicz** [erdewit/HiFiScan#18]
I've run into this issue on two separate Windows systems (10 & 11). Do you have an idea what could be causing this? Thanks!

```
C:\Users\Pc>python -m hifiscan
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\Pc\AppData\Local\Programs\Python\Python311\Lib\site-packages\hifiscan\__main__.py", line 4, in <module>
    main()
  File "C:\Users\Pc\AppData\Local\Programs\Python\Python311\Lib\site-packages\hifiscan\app.py", line 683, in main
    app = App()
          ^^^^^
  File "C:\Users\Pc\AppData\Local\Programs\Python\Python311\Lib\site-packages\hifiscan\app.py", line 46, in __init__
    self.resetAudio()
  File "C:\Users\Pc\AppData\Local\Programs\Python\Python311\Lib\site-packages\hifiscan\app.py", line 95, in resetAudio
    defaultRate = sd.query_devices(device='default')['default_samplerate']
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Pc\AppData\Local\Programs\Python\Python311\Lib\site-packages\sounddevice.py", line 566, in query_devices
    device = _get_device_id(device, kind, raise_on_error=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Pc\AppData\Local\Programs\Python\Python311\Lib\site-packages\sounddevice.py", line 2799, in _get_device_id
    raise ValueError(
ValueError: No input/output device matching 'default'
```

**Comment by erdewit** [comment]
Thanks for the bug report. It seems that a default device is not present on some systems. This has been addressed in v1.5.1.

**Comment by lukaszliniewicz** [comment]
Thank you for handling this so quickly! Unfortunately, I get another error after upgrading:

```
C:\Users\Pc>python -m hifiscan
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\Pc\AppData\Local\Programs\Python\Python311\Lib\site-packages\hifiscan\__main__.py", line 4, in <module>
    main()
  File "C:\Users\Pc\AppData\Local\Programs\Python\Python311\Lib\site-packages\hifiscan\app.py", line 685, in main
    app = App()
          ^^^^^
  File "C:\Users\Pc\AppData\Local\Programs\Python\Python311\Lib\site-packages\hifiscan\app.py", line 46, in __init__
    self.resetAudio()
  File "C:\Users\Pc\AppData\Local\Programs\Python\Python311\Lib\site-packages\hifiscan\app.py", line 97, in resetAudio
    defaultRate = defaultDevice.get('default_samplerate')
                  ^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'get'
```
